tools/loadtest/metrics/collect-metrics.sh

```
@@ -395,6 +434,91 @@ collect_ecs_utilization() {
+        | select((.stoppedAt | .[0:19]) <= ($e | .[0:19]))
+        | select([.containers[]? | select(.exitCode > 0)] | length > 0)] | length')
+  fi
+
+  local running running_arns
+  running=$(aws ecs list-tasks --cluster "$cluster" --service-name "$service" --desired-status RUNNING \
+    --region "$REGION" --output json 2>/dev/null || echo '{"taskArns":[]}')
+  running_arns=$(echo "$running" | jq -r '.taskArns[]')
+
+  details='{"tasks":[]}'
+  if [[ -n "$running_arns" ]]; then
+    details=$(aws ecs describe-tasks --cluster "$cluster" --tasks $running_arns \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)